lib/rubocop/cop/netlify/require_scope_confusion.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative "require_scope_base"
+
+module RuboCop
+  module Cop
+    module Netlify
+      # This cop checks OAuth scope definition duplication
+      #
+      # @example
+      #   # bad
+      #   require_scope "all:read"
+      #   require_scope "public"
+      #
+      #   # good
+      #   require_scope "public", "all:read"
+      #
+      #   # bad
+      #   require_scope "all:read", only: :index
+      #   require_scope "all:read", only: [:index, :show]
+      #
+      #   # good
+      #   require_scope "all:read", only: [:index, :show]
+      #
+      #   # bad
+      #   require_scope "all:read"
+      #   require_scope "all:write"
+      #
+      #   # good
+      #   require_scope "??" # Be careful! 
+      class RequireScopeConfusion < RequireScopeBase
```

Comment:
I'd name this `RequireScopeDuplication` to be extra clear, as "confusion" it's not very specific to this case.

Reply:
Done.